recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -104,6 +124,12 @@ def generate(self):
             tc.variables["BTCPP_UNIT_TESTS"] = False
             tc.variables["BTCPP_BUILD_TOOLS"] = self.options.with_tools
             tc.variables["BTCPP_ENABLE_COROUTINES"] = self.options.with_coroutines
+            tc.variables["BTCPP_GROOT_INTERFACE"] = self.options.with_groot_interface
+            tc.variables["BTCPP_SQLITE_LOGGING"] = self.options.with_sqlite_logging
+
+            if Version(self.version) == "4.0.1" or Version(self.version) == "4.0.2":
+                tc.variables["BTCPP_MANUAL_SELECTOR"] = False
+
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

Comment:
This can be dropped if the conan required version is bump to 1.54.0 :)

Reply:
You're right